lib/chef/provider/service/windows.rb

```diff
@@ -91,7 +91,7 @@ def start_service
       }.reject { |k, v| v.nil? || v.length == 0 }
 
       Win32::Service.configure(new_config)
-      logger.info "#{@new_resource} configured with #{new_config.inspect}"
```

Comment:
Seems like we might want want to emit the full message in debug assuming sensitive isn't set? We probably never needed it in info.

Reply:
I don't see much point in logging anything here at all, but as the existing config/reconfig was logging to `info` I figured it would be an unexpected change to do anything other than show the non-sensitive info (which amounts to just the resource name).  So I figured I'd just remove that possibly sensitive info.